tsconfig.json

```diff
@@ -25,7 +25,12 @@
     "isolatedModules": true,
 
     // we don't create declaration files when linting
-    "noEmit": true
+    "noEmit": true,
+    "baseUrl": ".",
+    "paths": {
```

Comment:
These alias are only for tests. Should we create a special config only for testing?

Reply:
Yes I think that would be better, I guess you could inherit from the "common" configuration and extend it with the jest specific paths without duplicating any config?

How would you then tell Jest to use it's custom tsconfig?